db/migrate/20250925094123_discard_mat_view_reporting_patient_states.rb

```diff
@@ -0,0 +1,849 @@
+class DiscardMatViewReportingPatientStates < ActiveRecord::Migration[6.1]
+  def change
+    # create_view :reporting_patient_states
+    drop_view :reporting_quarterly_facility_states, materialized: true
+    drop_view :reporting_facility_states, materialized: true
+    drop_view :reporting_overdue_patients, materialized: true
+    drop_view :reporting_facility_monthly_follow_ups_and_registrations, materialized: true
+    drop_view :reporting_patient_states, materialized: true
+    execute <<~SQL
+      CREATE VIEW public.reporting_patient_states AS SELECT * FROM simple_reporting.reporting_patient_states;
```

Comment:
I'm worried about this bit. A view is virtual i.e. doesn't store anything on disk. This means the query to produce the view would be run every time this view is accessed. There's no way the query for the dashboard would load within the request lifecycle 😅 

Can we keep this as a materialized view, but deprecate it somehow. A materialized view would store the data on disk for faster retrieval. We can just choose to not update it as frequently as before, or not update it at all. But to remove it so we run the query in the request lifecycle may be costly for environments with plenty data